<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>830</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Парсер законопроектів</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QGroupBox{border: &quot;0&quot;}
QTextEdit{font: 14pt &quot;MS Shell Dlg 2&quot;;}

</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1100</width>
      <height>830</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="bill_search_parameters_tab">
     <attribute name="title">
      <string>Параметри пошуку</string>
     </attribute>
     <widget class="QGroupBox" name="bill_search_parameters_groupBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>35</y>
        <width>780</width>
        <height>280</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="bill_search_parameters_verticalLayout">
       <property name="leftMargin">
        <number>11</number>
       </property>
       <item>
        <widget class="QGroupBox" name="session_groupBox">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <widget class="QLabel" name="session_label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>200</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сесія реєстрації:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QComboBox" name="session_comboBox">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>0</y>
            <width>200</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>Всі сесії IX скл.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IX скл. 9 сесія</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IX скл. 8 сесія</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IX скл. 7 сесія</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IX скл. 6 сесія</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IX скл. 5 сесія</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IX скл. 4 сесія</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IX скл. 3 сесія</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IX скл. 2 сесія</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IX скл. 1 сесія</string>
           </property>
          </item>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="registration_number_groupBox">
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <widget class="QLabel" name="registration_n_label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>200</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;№ реєстрації:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QComboBox" name="registration_number_compare_operation_comboBox">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>0</y>
            <width>200</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <item>
           <property name="text">
            <string>Містить</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Дорівнює</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Перелік</string>
           </property>
          </item>
         </widget>
         <widget class="QTextEdit" name="registration_number_textEdit">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>0</y>
            <width>200</width>
            <height>40</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>1</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="registration_date_groupBox">
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="registration_date_label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>200</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Дата реєстрації:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="registration_range_start_dateEdit">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>0</y>
            <width>200</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2022</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="date">
           <date>
            <year>2022</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
         <widget class="QDateEdit" name="registration_range_end_dateEdit">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>0</y>
            <width>200</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="date">
           <date>
            <year>2023</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="name_groupBox">
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="name_label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>270</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Назва (слова з назви):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="name_textEdit">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>0</y>
            <width>440</width>
            <height>40</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="start_parsing_pushButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>700</y>
        <width>431</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bill_parse_parameters_tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Параметри парсингу</string>
     </attribute>
     <widget class="QGroupBox" name="bill_parse_parameters_groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1090</width>
        <height>800</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="fields_to_parse_groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>230</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox{font: 12pt &quot;MS Shell Dlg 2&quot;;}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="fields_to_parse_lable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>230</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Поля для парсингу</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="fields_to_parse_verticalGroupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>0</width>
          <height>0</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="fields_to_parse_verticalLayout"/>
       </widget>
      </widget>
      <widget class="QGroupBox" name="download_parameters_groupBox">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>15</y>
         <width>460</width>
         <height>140</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="download_parameters_label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>0</y>
          <width>300</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Файли для скачування</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="download_parameters_verticalGroupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>471</width>
          <height>85</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="governing_documents_checkBox">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Текст законопроекту та супровідні документи</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="related_to_work_documents_checkBox">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Документи, пов'язані із роботою</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="file_format_parameters_verticalGroupBox">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>670</y>
         <width>550</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="file_format_label_9">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Формат:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="xlsx_file_format_radioButton_9">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>xlsx</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="csv_file_format_radioButton_9">
          <property name="text">
           <string>csv</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>json</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="directory_verticalGroupBox">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>600</y>
         <width>500</width>
         <height>57</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="directory_pushButton">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Директорія</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="directrory_textEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
